### DISCUSSION TITLE
Can emqx clusters be automatically discovered across different network segments?

### DISCUSSION THREAD

**Discussion by flex380** [emqx/emqx#5688] [Q&A]
such as:
node1:
emqx1@192.168.0.30
node2:
emqx2@192.168.1.30

Or whether it can be realized by proxy?

**Comment by k32** [comment]
Hello,
It depends on the discovery strategy, EMQ X supports have several: https://docs.emqx.io/en/broker/v4.3/getting-started/cluster.html#node-discovery-and-autocluster
etcd and k8s strategies support this.
